Src/WitsmlExplorer.Api/Services/CredentialsCache.cs

```diff
@@ -8,11 +8,12 @@ namespace WitsmlExplorer.Api.Services
 
     public interface ICredentialsCache
```

Comment:
Being a three level dictionary, I think we could consider outlining the cache setup. If you agree we could perhaps add that as a <summary> over this class. 
(Something similar to)
```
{ "cacheId" : {
    "serverUrl.Host": {
      "username": "encrypted_password"
    }
  }
}
```